extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityTodoListModel.class.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class CommunityPageSpecialCommunityTodoListModel {
+	public function getData() {
+		$title = Title::newFromText(
+			wfMessage( 'communitypage-todo-module-page-name' )->inContentLanguage()->plain(),
```

Comment:
Why not just make use of `wfMessage` and parse the contents with an output format? Also, having a customisable page name for this functionality will be difficult as each permutation would need to be whitelisted so admins could edit it. For this purpose, I'd suggest just making `Community-to-do-list` the page name and not have another message control that (MediaWiki pages have always been in English anyway).

Then, this method could just be something like:
```php
$message = wfMessage( 'community-to-do-list' );
$data = [
	'haveContent' => false,
	'editUrl' => $editUrl,
];

if ( !$message->isDisabled() ) {
	$data['haveContent'] = true;
	$data['data'] = $message->parse();
}

return $data;
```